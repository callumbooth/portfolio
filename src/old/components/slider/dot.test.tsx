import React from 'react';

import { render, screen } from '@testing-library/react';

import Dot from './dot';

describe('Dot', () => {
    it('Dot displays when inactive', () => {
        const component = render(
            <Dot active={false} count={0} handleClick={jest.fn()} />,
        );

        expect(component.container).toMatchSnapshot();
    });

    it('Dot displays when active', () => {
        render(<Dot active={true} count={1} handleClick={jest.fn()} />);

        expect(screen.getByLabelText('dot-1')).toBeTruthy();
    });
});
