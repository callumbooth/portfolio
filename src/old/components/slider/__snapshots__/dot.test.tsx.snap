// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dot Dot displays when inactive 1`] = `
<div>
  <button
    aria-label="dot-0"
    class="dot rounded bg-gray-300 border border-solid border-primary-main w-5 h-5 mx-4 cursor-pointer transition duration-200 hover:bg-primary-main active:bg-primary-main"
    data-target="1"
    id="dot-0"
  />
</div>
`;
